packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-venmo-button-strategy.spec.ts

```diff
@@ -52,7 +51,7 @@ describe('PaypalCommerceVenmoButtonStrategy', () => {
     const defaultButtonContainerId = 'paypal-commerce-venmo-button-mock-id';
     const approveDataOrderId = 'ORDER_ID';
 
-    const paypalCommerceVenmoOptions: PaypalCommerceVenmoButtonInitializeOptions = {
+    const paypalCommerceVenmoOptions = {
```

Comment:
Strange that we needed to remove type here ?

Reply:
We should remove the type here due to the interface conflict that we have in another PR:
https://github.com/bigcommerce/checkout-sdk-js/pull/1813